en/config_heli/README.md

```diff
@@ -0,0 +1,89 @@
+# Helicopter Configuration
+
+This section contains topics related to helicopter configuration and tuning.
+
+:::note
+Helicopters are less well-supported than other types of vehicles.
+We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
```

Comment:
With respect to what this means then https://github.com/PX4/PX4-user_guide/pull/2216/files#r1059837623, how about:

```suggestion
PX4 supports helicopters with coaxial or dual rotor types, and features such as RPM governor and autorotation. It can generally be used for the same kinds of things as other multicopters.

We would welcome your [contribution](../contribute/README.md) of new features, new frame configurations, or other improvements.
```

Reply:
PS FWIW one day it would be good to have a helicopter introduction page that covers the supported features etc - i.e. that stuff ^^^. That could then be the parent for build logs and so on, in the same way as http://localhost:8080/px4_user_guide/en/frames_vtol/

Reply:
> PX4 supports helicopters with coaxial or dual rotor types

PX4 *does not support* ...